frontend/src/util/task-util.ts

```diff
@@ -237,16 +237,22 @@ function removeOneTimeTask(task: OneTimeTask): void {
     return false;
   });
   if (!isFork) {
-    removeTask(task);
-    return;
+    promptConfirm(
+      'Do you really want to remove this task? The removed task cannot be recovered.',
+    ).then((confirmed) => {
+      if (confirmed) {
+        removeTask(task);
+      }
+    });
+  } else {
+    promptConfirm(
+      'Do you really want to remove this forked task? The removed task cannot be recovered.',
+    ).then((confirmed) => {
+      if (confirmed) {
+        removeTask(task);
+      }
+    });
   }
```

Comment:
Is it necessary to have both branches of this if statement? Can we just have a single conditional to change the string passed to the dialog box?

Reply:
Then would the message be "Do you really want to remove this task? The removed task cannot be recovered." for both forked and non-forked tasks?